packages/node/test/setup/e2e/testing.ts

```diff
@@ -4,7 +4,7 @@ import { buildChainConfig, buildProvider, getDeployerIndex } from './utils';
 import { deployAirnodeRrp, makeRequests } from './deployment';
 import { buildConfig, operation } from '../../fixtures';
 
-export const increaseTestTimeout = (timeoutMs = 45_000) => jest.setTimeout(timeoutMs);
+export const increaseTestTimeout = (timeoutMs = 60_000) => jest.setTimeout(timeoutMs);
```

Comment:
Did I commit this by accident or this is something we want to update?

Reply:
Explicit choice by me: https://github.com/api3dao/airnode/commit/3ba5ae7e842d98c3bbd46485d296cbc6643a6be6

I am not sure if my machine is just slow, but it might be because of the recent changes? (The fact is it doesn't pass locally until I increase the timeout)